routetable/route_table.go

```diff
@@ -148,6 +165,8 @@ func (r *RouteTable) Apply() error {
 			}
 		}
 		r.inSync = true
+
+		listIfaceTime.Observe(time.Since(listStartTime).Seconds())
```

Comment:
No longer worried about backwards time jumps?

(Perhaps you're happy now that they don't actually occur in this interface.  That would be great if so!)